# Detect shipment
# Called to ask a tray to report what products were placed on it
---
# What was detected in the tray
DetectedKittingShipment shipment
